lib/src/main/java/tech/relaycorp/awaladroid/endpoint/FirstPartyEndpoint.kt

```diff
@@ -250,7 +248,7 @@ internal constructor(
             }
 
             context.storage.gatewayPrivateAddress.set(
```

Comment:
Just like we're saving the gateway's ~private address~ id, we should also be saving its Internet address (which is available in the `registration` object above).

This library won't use that value, but the apps that use this library will. For example, the Ping app has to send that value to its peer so that the peer can reply to it.